src/assets/i18n/en.json5

```diff
@@ -5639,4 +5639,332 @@
   "admin.notifications.publicationclaim.breadcrumbs": "Publication Claim",
 
   "admin.notifications.publicationclaim.page.title": "Publication Claim",
+
+  "coar-notify-support.title": "COAR Notify Protocol",
+
+  "coar-notify-support-title.content": "Here, we fully support the COAR Notify protocol, which is designed to enhance the communication between repositories. To learn more about the COAR Notify protocol, you can visit their official website <a href=\"https://notify.coar-repositories.org/\">here</a>.",
+
+  "coar-notify-support.ldn-inbox.title": "LDN InBox",
+
+  "coar-notify-support.ldn-inbox.content": "For your convenience, our LDN (Linked Data Notifications) InBox is easily accessible at {ldnInboxUrl}. The LDN InBox enables seamless communication and data exchange, ensuring efficient and effective collaboration.",
+
+  "coar-notify-support.message-moderation.title": "Message Moderation",
+
+  "coar-notify-support.message-moderation.content": "To ensure a secure and productive environment, all incoming LDN messages are moderated. If you are planning to exchange information with us, kindly reach out via our dedicated Feedback form. You can access the Feedback form by clicking <a href=\"info/feedback\">here</a>.",
```

Comment:
Same thing. Never have the text "here" be a link, as it doesn't describe the link.  We should update this to add the link to the text "Feedback form".  We also may need to change this so that it's not an `href`, as clicking a link like this will cause the application to reload entirely...it triggers SSR.